dev-util/bakefile/patches/bakefile-0.2.11.patchset

```diff
@@ -0,0 +1,91 @@
+From 86a685089b703cd5d5747e897c69aae3cc2e6781 Mon Sep 17 00:00:00 2001
+From: begasus <begasus@gmail.com>
+Date: Mon, 21 May 2018 15:23:44 +0200
+Subject: import previous patch
+
+
+diff --git a/autoconf/bakefile.m4 b/autoconf/bakefile.m4
+index efa98ae..e8f771d 100644
+--- a/autoconf/bakefile.m4
++++ b/autoconf/bakefile.m4
+@@ -73,6 +73,7 @@ AC_DEFUN([AC_BAKEFILE_PLATFORM],
+     PLATFORM_MACOS=0
+     PLATFORM_MACOSX=0
+     PLATFORM_BEOS=0
++    PLATFORM_HAIKU=0
+ 
+     if test "x$BAKEFILE_FORCE_PLATFORM" = "x"; then
+         case "${BAKEFILE_HOST}" in
+@@ -86,6 +87,9 @@ AC_DEFUN([AC_BAKEFILE_PLATFORM],
+             *-*-beos* )
+                 PLATFORM_BEOS=1
+             ;;
++            *-*-haiku* )
++                PLATFORM_HAIKU=1
++            ;;
+             powerpc-apple-macos* )
+                 PLATFORM_MAC=1
+                 PLATFORM_MACOS=1
+@@ -109,6 +113,9 @@ AC_DEFUN([AC_BAKEFILE_PLATFORM],
+             beos )
+                 PLATFORM_BEOS=1
+             ;;
++            *-*-haiku* )
++                PLATFORM_HAIKU=1
++            ;;
+             * )
+                 AC_MSG_ERROR([Unknown platform: $BAKEFILE_FORCE_PLATFORM])
+             ;;
+@@ -121,6 +128,7 @@ AC_DEFUN([AC_BAKEFILE_PLATFORM],
+     AC_SUBST(PLATFORM_MACOS)
+     AC_SUBST(PLATFORM_MACOSX)
+     AC_SUBST(PLATFORM_BEOS)
++    AC_SUBST(PLATFORM_HAIKU)
+ ])
+ 
+ 
+@@ -146,7 +154,7 @@ AC_DEFUN([AC_BAKEFILE_PLATFORM_SPECIFICS],
+         fi
+         ;;
+ 
+-      i*86-*-beos* )
++      i*86-*-beos* | i*86-*-haiku*)
+         LDFLAGS="-L/boot/develop/lib/x86 $LDFLAGS"
```

Comment:
didn't make a noice when building, I left the 0.2.8 in because it still builds on x86_gcc2

Reply:
PS, is there a way to put a variable there instead of a hard path?

Reply:
Maybe something like 379f6106ec0ff2112539d1b2fd32a03b76f7fdf0.
But it might even be possible to use environment variables and pass them at configure and/or build time, instead of adding a `PATCH()` function.

Reply:
Maybe something like this could be used? 
https://github.com/haikuports/haikuports/blob/75a3cf5384f650242f4b8a1094ab305d33ed99e7/app-cdr/cdrtools/patches/cdrtools-3.02-Defaults.haiku.patch#L9

Reply:
Yeah, this looks great (and I think it will work.)

Reply:
I would recommend using `findpaths B_FIND_PATH_DEVELOP_LIB_DIRECTORY`, as it will list all `develop/lib` directory (including `non-packaged`, `user/config`). Some bash-fu would be needed to convert the result into `-L`s however.

Reply:
I got it :)

Reply:
I'll see when I get around to it to test it :)

Reply:
Used the finddir from cdrtools for now, my bash-fu isn't that skilled to use the one suggested by alaviss :)

Reply:
I'm OK with this :)
@alaviss, are you OK with giving up the bash-fu for now?

Reply:
Sure :) I will fix it myself later when I've time ;)